orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationJacksonConfiguration.java

```diff
@@ -0,0 +1,49 @@
+package com.sap.ai.sdk.orchestration;
+
+import static com.sap.ai.sdk.core.JacksonConfiguration.getDefaultObjectMapper;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.google.common.annotations.Beta;
+import com.sap.ai.sdk.orchestration.model.ChatMessagesInner;
+import com.sap.ai.sdk.orchestration.model.LLMModuleResult;
+import com.sap.ai.sdk.orchestration.model.ModuleResultsOutputUnmaskingInner;
+import javax.annotation.Nonnull;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.val;
+
+/** Internal utility class for getting a default object mapper with preset configuration. */
+@NoArgsConstructor(access = AccessLevel.NONE)
+public class OrchestrationJacksonConfiguration {
```

Comment:
(Major)

You are publishing this API.
Please add `@Beta`
Please add `@since` in JavaDoc